x-pack/plugins/transform/public/plugin.ts

```diff
@@ -9,6 +9,7 @@ import { i18n as kbnI18n } from '@kbn/i18n';
 
 import type { CoreSetup } from '@kbn/core/public';
 import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+import { DataViewsPublicPluginStart } from '@kbn/data-views-plugin/public';
```

Comment:
```suggestion
import type { DataViewsPublicPluginStart } from '@kbn/data-views-plugin/public';
```

Reply:
Done in https://github.com/elastic/kibana/pull/131819/commits/36d639885f4609acb2982b083b9639e7a47b7956